locale/zh-cn/docs/guides/dont-block-the-event-loop.md

```diff
@@ -256,19 +256,19 @@ app.get('/redos-me', (req, res) => {
 var obj = { a: 1 };
 var niter = 20;
 
-var before, res, took;
+var before, str, pos, res, took;
 
 for (var i = 0; i < niter; i++) {
-  obj = { obj1: obj, obj2: obj }; // 每个循环里面将对象 size 加倍
+  obj = { obj1: obj, obj2: obj }; // Doubles in size each iter
```

Comment:
The comments could stay since they are translated.

Reply:
Fixed.